src/main/resources/application-mysql.properties

```diff
@@ -1,7 +1,7 @@
 # database init, supports mysql too
 database=mysql
-spring.datasource.url=jdbc:mysql://localhost/petclinic
-spring.datasource.username=root
-spring.datasource.password=petclinic
-# SQL is written to be idempotent so this is safe
+spring.datasource.url=${MYSQL_DB:jdbc:mysql://localhost/petclinic}
+spring.datasource.username=${MYSQL_USER:petclinic@petclinicdb2}
```

Comment:
Not sure we want the default to be `@anything`. The default should be fine without it.